pkg/cli/rm_helper.go

```diff
@@ -0,0 +1,207 @@
+package cli
+
+import (
+	"fmt"
+	"github.com/acorn-io/acorn/pkg/prompt"
+	"github.com/pterm/pterm"
+	"strings"
+
+	"github.com/acorn-io/acorn/pkg/client"
+	"github.com/spf13/cobra"
+)
+
+func addRmObject(rmObjects *RmObjects, obj string) {
+	switch strings.ToLower(obj) {
+	case "app":
+		rmObjects.App = true
+	case "container":
+		rmObjects.Container = true
+	case "secret":
+		rmObjects.Secret = true
+	case "volume":
+		rmObjects.Volume = true
+	case "a":
+		rmObjects.App = true
+	case "c":
+		rmObjects.Container = true
+	case "s":
+		rmObjects.Secret = true
+	case "v":
+		rmObjects.Volume = true
+	default:
+		pterm.Warning.Printf("%s is not a valid type\n", obj)
+	}
+}
+func getSecretsToRemove(arg string, client client.Client, cmd *cobra.Command) ([]string, error) {
```

Comment:
minor style nit: i really prefer functions have a blank line between the end of one and start of the next